templates/2.1/mapping.json

```diff
@@ -0,0 +1,838 @@
+{
+    "version": "2.1",
+    "components": {
+        "agent": {
+            "version": {
+                "section": "agent",
+                "option": "version",
+                "default": "2.1"
+            },
+            "revocation_notification_ip": {
```

Comment:
I think someone should still look at the diff between the 2.0 mapping.json and this one. There seem to be lots of changes unrelated to IDevID and IAK support and I don't know as much about these files.

Reply:
I'll check them

Reply:
The changes LGTM. The part that may be confusing is that the `mapping.json` is now mapping the sections and options from version `2.0` to the options in version `2.1`, and not from the legacy `1.0` (when we had a single `keylime.conf` file).

With the new feature I'm adding in #1486 this kind of change will be much smaller in future as only the differences will show up in the mappings updates.